src/backend/impl/src/services/proposal_review_service.rs

```diff
@@ -407,6 +453,33 @@ impl<
 
         Ok(())
     }
+
+    fn validate_published_fields(
+        &self,
+        summary: Option<&String>,
+        review_duration_mins: Option<u16>,
+        build_reproduced: Option<bool>,
+    ) -> Result<(), ApiError> {
+        if summary.is_none() {
+            return Err(ApiError::invalid_argument("Summary cannot be empty"));
+        }
+
+        if review_duration_mins.is_none() {
+            return Err(ApiError::invalid_argument(
+                "Review duration cannot be empty",
+            ));
+        }
+
+        if build_reproduced.is_none() {
+            return Err(ApiError::invalid_argument(
+                "Build reproduced cannot be empty",
+            ));
+        }
```

Comment:
I think we should cover these cases in the integration tests, since before they were always passing due to the default values

Reply:
Sure, I'll add those tests.

Reply:
Added now 👍

Reply:
Yep saw it! I like the `it.each` test organization